content/en/blog/2022-03-21-tools-to-manage-helm-declaratively.md

```diff
@@ -47,38 +47,40 @@ The scope of this section is limited to [graduated and incubating](https://www.c
 There are many projects beyond the CNCF projects you can use to help you manage your Helm releases. The following set is an example and not exhaustive.
 
 * [Helmfile](https://github.com/roboll/helmfile) - A declarative spec for deploying Helm charts.
-* [Captain](https://github.com/alauda/captain) - A Helm controller
-* [Helm Provider for Terraform](https://github.com/hashicorp/terraform-provider-helm) - Enables you to manage Helm charts through Terraform
+* [Captain](https://github.com/alauda/captain) - A Helm controller.
+* [Terraform Helm provider](https://github.com/hashicorp/terraform-provider-helm) - Enables you to manage Helm charts through Terraform.
 * [Orkestra](https://azure.github.io/orkestra/) - Built on other tools in this list, Orkestra adds a robust dependency graph for a related group of Helm releases and their subcharts, as well as a reverse DAG for specifying dependency requirements for rollbacks.
 * [Fleet](https://github.com/rancher/fleet) - A GitOps tool chain that works with Kubernetes manifests, Helm charts, and Kustomize.
 
 ### High-Level Tool Comparison
 
 There are some differences between the tools we've looked at so far. The following table provides some insight into their differences. This is not exhaustive and you should evaluate any tools you use yourself.
 
-| | Retains Helm release info | Supports Helm hooks | OCI support |
-| -- | -- | -- | -- |
-| Helmfile | ✅ | ❓[^1] | 🚫 |
-| Terraform Helm provider | ✅ | ✅[^2] | 🚫[^3] |
-| Captain Helm controller | ✅ | ❓[^4] | 🚫[^5] |
-| Flux Helm controller | ✅ | ✅  | 🚫[^6] |
-| ArgoCD | 🚫 | 🚫[^7] | ✅[^8] |
-| Orkestra | ✅ | ✅ | 🚫[^9] |
-| Fleet | ✅ | ✅ | 🚫[^10] |
+| | Retains Helm release info | Supports Helm hooks | OCI support | Does not require Helm binary |
+| -- | -- | -- | -- | -- |
+| Flux Helm controller | ✅ | ✅ | ✅[^1] | ✅ |
```

Comment:
@yurrriq ah I missed this before merging – the Flux controller 3rd column should still be 🚫 because it is not yet implemented, even though it's well on it's way (explained in comment). I wouldn't want to mislead people at a glance in the table. I will fix on my PR branch for https://github.com/helm/helm-www/pull/1293

Reply:
Thanks. Probably a mistake when I was rewriting history.